travis_ci_flake8.cfg

```diff
@@ -106,14 +100,17 @@ extend-ignore =
     # W605: invalid escape sequence '\d'
     W605,
 per-file-ignores =
```

Comment:
the commas at end of line don't really belong there because this is a mapping (`... : ...`) rather than a list (as above)